packages/ui-extensions/src/surfaces/point-of-sale/types/cart.ts

```diff
@@ -107,3 +108,84 @@ export interface Address {
   provinceCode?: string;
   countryCode?: CountryCode;
 }
+
+export type ShippingLine =
+  | {
+      handle: string;
+      price: string;
+      title: string;
+      methodType: ShippingMethodType;
+      type: 'Calculated';
```

Comment:
Should we move these two into two interfaces `CalculatedShippingLine` and `CustomShippingLine` and then just make `ShippingLine = CalculatedShippingLine | CustomShippingLine` ?